cpp/src/arrow/util/fixed_width_test_util.h

```diff
@@ -0,0 +1,203 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Can this file please be moved into `arrow/testing`, since those are testing utilities?

Reply:
Opened #41558